.github/CODEOWNERS

```diff
@@ -30,8 +27,6 @@
 /corehq/apps/callcenter/ @snopoke
 /corehq/apps/change_feed/ @dannyroberts
 /corehq/apps/commtrack/ @esoergel
-/corehq/apps/commtrack/static/ @esoergel @orangejenny
```

Comment:
Was it intentional that you also removed @esoergel?

Reply:
yes, he's still covered by the line right above this

Reply:
Obviously.